scripts/build.sh

```diff
@@ -158,22 +165,25 @@ build() {
         version=$(grep -i version "$controlFile" | awk '{ print $2 }')
         echo "Extracting contents to tar file"
         ./scripts/extract-scripts.sh "$pkgFolder".deb
-        filename="${pkg}_${version}_$PLATFORM".tar
+        filename="${pkg}_${version}_$PLATFORM$tag".tar
         mv "$pkgFolder".deb.tar "$cwd/$filename"
 
     fi
+    ls -l "$cwd/$filename"
 
 }
 
 buildDeb=0
 arch="armhf"
+tag=""
```

Comment:
Is there a way for us to not repeat all these empty `tag=""`? Could we instead default to an empty string if the arg isn't present?

Reply:
Well I thought about that, however each group fully defines the variables needed prior to calling "build()".  I realize that as written, that appears redundant however the intent is to make clear that "tag" needs to be set.  My fear is that someone adding a new build target might not notice it if it was just set once and assumed to apply for all the subsequent build() calls.  There's probably a better way to refactor the whole build script to make all of these things cmd line params of the build call.

Reply:
I guess what I am saying is why not just make it optional/default to `""`?

Reply:
Fixed with additional changes below.